cypress/integration/user_profile/BlockUser.feature

```diff
@@ -0,0 +1,46 @@
+Feature: Block a User
+  As a user
+  I'd like to have a button to block another user
+  To prevent him from seeing and interacting with my contributions
+
+  Background:
+    Given I have a user account
+    And there is an annoying user called "Harassing User"
+    And I am logged in
+
+  Scenario: Block a user
+    Given I am on the profile page of the annoying user
+    When I click on "Block user" from the content menu in the user info box
+    And I navigate to my "Blocked users" settings page
+    Then I can see the following table:
+      | Avatar | Name           |
+      |        | Harassing User |
+
+  Scenario: Blocked user cannot interact with my contributions
+    Given I block the user "Harassing User"
+    And I previously created a post
+    And a blocked user visits the post page of one of my authored posts
+    Then they should not see the comment from
+    And they should see a text explaining commenting is not possible
+
+  Scenario: Block a previously followed user
+    Given I follow the user "Harassing User"
+    When I visit the profile page of the annoying user
+    And I click on "Block user" from the content menu in the user info box
+    And I get removed from his follower collection
```

Comment:
> [<img alt="roschaefer" height="40" width="40" align="left" src="https://avatars1.githubusercontent.com/u/38261864?s=88&v=4">](https://github.com/roschaefer) **Authored by [roschaefer](https://github.com/roschaefer)**
_<time datetime="2020-01-23T17:10:06Z" title="Thursday, January 23rd 2020, 6:10:06 pm +01:00">Jan 23, 2020</time>_

---
> **Outdated (history rewrite)** - original diff
---
```diff
@@ -0,0 +1,48 @@
+Feature: Block a User
+  As a user
+  I'd like to have a button to block another user
+  To prevent him from seeing and interacting with my contributions
+
+  Background:
+    Given I have a user account
+    And there is an annoying user called "Harassing User"
+    And I am logged in
+
+  Scenario: Block a user
+    Given I am on the profile page of the annoying user
+    When I click on "Block user" from the content menu in the user info box
+    And I navigate to my "Blocked users" settings page
+    Then I can see the following table:
+      | Avatar | Name           |
+      |        | Harassing User |
+
+  Scenario: Blocked user cannot interact with my contributions
+    Given I block the user "Harassing User"
+    And I previously created a post
+    And the blocked user visits my post
+    Then they should not see the comment from
+    And they should see a text explaining commenting is not possible
+
+  Scenario: Block a previously followed user
+    Given I follow the user "Harassing User"
+    When I visit the profile page of the annoying user
+    And I click on "Block user" from the content menu in the user info box
+    And nobody is following the user profile anymore
```


```suggestion
    Then I get removed from his follower collection
```

Reply:
> [<img alt="mattwr18" height="40" width="40" align="left" src="https://avatars1.githubusercontent.com/u/38261864?s=88&v=4">](https://github.com/mattwr18) **Authored by [mattwr18](https://github.com/mattwr18)**
_<time datetime="2020-01-23T17:43:24Z" title="Thursday, January 23rd 2020, 6:43:24 pm +01:00">Jan 23, 2020</time>_

---


hahaha... it was you who wrote this, but I'll update it